MarkBind/markbind
# Fix backslash bugs in convert feature

**What is the purpose of this pull request?**


Fixes #1736 


**Overview of changes:**

Updated the relativePagePath to ensure that no unnecessary `\` were being made. Done by updating a function from fsUtil

**Anything you'd like to highlight / discuss:**


**Testing instructions:**



Run `markbind init -convert` on a test repo and make sure there are no instances of `\` or `%5C` in the `layouts/default.md` file.
Build and see that there also should not have any instance of `\` or `%5C`


**Proposed commit message: (wrap lines at 72 characters)**

Resolved bug regarding backslashes in convert feature



---

**Checklist:** :ballot_box_with_check:



- [ ] Added tests for bug fixes or features (will be done in another PR)
